google-cloud-compat-checker/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-compat-checker</artifactId>
+  <version>0.21.2-alpha-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Java Compatibility Checker</name>
+  <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-compat-checker</url>
+  <description>
+    Compatibility checker for Google Cloud Java.
+  </description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-pom</artifactId>
+    <version>0.21.2-alpha-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <site.installationModule>google-cloud-compat-checker</site.installationModule>
+    <artifact.version>${project.version}</artifact.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>2.0.3.Final</version>
+    </dependency>
+    <dependency>
+      <!-- It may look weird to have a compile dependency on a plugin,
+	   but we want to execute code from it at runtime, in order
+	   to not duplicate the logic ourselves -->
+      <groupId>kr.motd.maven</groupId>
```

Comment:
Do we really need this strange dependency? Usually it is possible to get enough of information about the running environment by means of core java with combination of environment variables and system properties, plus, maybe some more advanced (but still built in stuff). For example:

```java
PrintStream out = new PrintStream(System.out);

out.append("\n\nSYSTEM PROPERTIES:\n\n");
System.getProperties().list(out);

//out.append("\nENVIRONMENT VARIABLES:\n\n");
//Map<String, String> envVars = System.getenv();
//for (Map.Entry<String, String> entry : envVars.entrySet()) {
//  out.append(entry.getKey()).append("=").append(entry.getValue()).append('\n');
//}

```

Among other garbage, that outputs stuff like:
```
os.name=Linux
os.version=4.4.0-83-generic
os.arch=amd64

```